packages/shared/src/components/drawers/Drawer.tsx

```diff
@@ -35,7 +40,7 @@ interface DrawerProps
   isFullScreen?: boolean;
   isClosing?: boolean;
   title?: string;
-  onClose(): void;
+  onClose(e: MouseEvent | KeyboardEvent | MessageEvent): void;
```

Comment:
You might see a lot of these changes, this is due to Drawer's requirement to fulfill outside click, it uses the native event handler.